dataset/dictionary/locations.json5

```diff
@@ -186,7 +186,7 @@
     tags: [ "mondstadt", "location" ],
   },
   {
-    en: "Stormterror's Liar",
+    en: "Stormterror's Lair",
```

Comment:
Oops, this was a typo.

When English is changed, the permalink for this word is changed at the same time.
Can you add the redirect setting to [redirect/words.json](https://github.com/xicri/genshin-langdata/blob/main/dataset/redirect/words.json)?

Current URL is https://genshin-dictionary.com/ja/stormterrors-liar/, so you can configure the redirect by adding the following line:

```json
"stormterrors-liar": "stormterrors-lair"
```

Reply:
Just add this line after `tags: [ "..." ]` right? I will do it right now.

Reply:
Ah, no, you need to add the line in another file: [redirect/words.json](https://github.com/xicri/genshin-langdata/blob/main/dataset/redirect/words.json)

Reply:
Oh, OK.